.travis.yml

```diff
@@ -0,0 +1,16 @@
+language: go
+
+go:
+  - 1.5.1
+  - 1.4.3
+  - 1.3.3
+
+before_install:
+  - go get golang.org/x/tools/cmd/vet
+# maybe one day
+#  - go get github.com/golang/lint/golint
+
+sudo: false
+
+script:
+  - go vet -x ./...
```

Comment:
i guess we need to ensure openssl devel is present so that we can `go test ./...` ?